ext/jinja2/src/main/resources/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think you can remove this, as the top level project contains the Apache in its LICENSE file.